src/plugins/intel_gna/gna_plugin.cpp

```diff
@@ -720,6 +721,7 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
         manager.register_pass<BroadcastAddMultiplyConst>();
         // UnrollTI should be the last transformation in the transformation pipeline
         manager.register_pass<ngraph::pass::UnrollTensorIterator>();
+        manager.register_pass<TransposeToPwl>(config.gnaFlags.pwlMaxErrorPercent);
```

Comment:
All deprecated options should still supported (uniformPwlDesign)

Reply:
done